website_logo/README.rst

```diff
@@ -1,21 +1,37 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
+============
 Website logo
 ============
 
-Load a logo image to be used on website only. This allows to use an internal company logo (for reports) and a different website logo.
-
+Load a logo image to be used on website only. This allows to use an internal
+company logo (for reports) and a different website logo.
 
 Configuration
 =============
 
 To configure this module, you need to:
 
-* go to your company and load a logo image in the website logo field
+* Go to 'Website Admin'.
+* In the configuration form of a website there is a category logo.
+* Load a logo image in the logo field.
+
+Usage
+=====
+
+This addon supports multi-website. If no logo is defined a transparent image is
```

Comment:
Yes, it's mandatory in English (http://www.shertonenglish.com/resources/es/conditionals/type0.php)